kubernetes/configs/es1-att/frontend.sh

```diff
@@ -6,6 +6,8 @@ PUBLIC_BUCKET_URL="https://interop-${ENVIRONMENT}-public${REGION_SUFFIX}.s3.${AW
 MIXPANEL_PROJECT_ID="fill_me"
 ONETRUST_DOMAIN_SCRIPT_ID="fill_me"
 SELFCARE_BASE_URL="https://selfcare.pagopa.it"
+API_SIGNAL_HUB_PUSH_INTEFACE_URL="https://raw.githubusercontent.com/pagopa/interop-signalhub-core/refs/heads/main/docs/openAPI/push-signals_1.0.0.yaml"
```

Comment:
```suggestion
API_SIGNAL_HUB_PUSH_INTERFACE_URL="https://raw.githubusercontent.com/pagopa/interop-signalhub-core/refs/heads/main/docs/openAPI/push-signals_1.0.0.yaml"
```
this typo is replicated in the other files as well 😄